HARDWARE.md

```diff
@@ -0,0 +1,13 @@
+# Bottlerocket test hardware
```

Comment:
I feel like this could be consolidated with the PROVISIONING-METAL doc, and I wonder if that one should be called QUICKSTART-METAL.

Reply:
IMHO, the hardware felt worthy of it's own doc rather than buried in other metal documentation.

Re: QUICKSTART-METAL, I opted not to create a "quickstart" type doc since provisioning bare metal machines isn't really prescriptive like we can be with AWS variants; "run this command to start an instance", etc.

Reply:
> IMHO, the hardware felt worthy of it's own doc rather than buried in other metal documentation.
>
> Re: QUICKSTART-METAL, I opted not to create a "quickstart" type doc since provisioning bare metal machines isn't really prescriptive like we can be with AWS variants; "run this command to start an instance", etc.

Makes sense. I'm still wondering if HARDWARE.md and PROVISIONING-METAL.md should be consolidated instead of being separate documents. Maybe METAL.md?